### PR TITLE
Moving Getting Started guide and adding redirects

### PR DESCRIPTION
**Pull request by runleonarun**
## Description & motivation

Pair writing session with @nghi-ly to move Getting Started guide and fix spacing in sidebar.

## To-do before merge
- [ ] Check redirects
- [ ] Confirm that new landing pages are OK
- [ ] Check for links that are broken

## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`
- [ ] [Run link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: _redirects [hunk 1]
```diff
@@ -463,3 +463,18 @@ docs/contributing/documenting-a-new-adapter /docs/contributing/adapter-developme
...
+/guides/getting-started/building-your-first-project/schedule-a-job /docs/develop/getting-started/building-your-first-project/schedule-a-job 301
+/guides/getting-started/building-your-first-project/test-and-document-your-project /docs/develop/getting-started/building-your-first-project/test-and-document-your-project 301
+/guides/getting-started/create-a-project /docs/develop/getting-started/create-a-project 301
+/guides/getting-started/getting-set-up /docs/develop/getting-started/getting-set-up 301
+/guides/getting-started/getting-set-up/setting-up-bigquery /docs/develop/getting-started/getting-set-up/setting-up-bigquery 301
+/guides/getting-started/getting-set-up/setting-up-databricks /docs/develop/getting-started/getting-set-up/setting-up-databricks 301
+/guides/getting-started/getting-set-up/setting-up-redshift /docs/develop/getting-started/getting-set-up/setting-up-redshift 301
+/guides/getting-started/getting-set-up/setting-up-snowflake /docs/develop/getting-started/getting-set-up/setting-up-snowflake 301
+/guides/getting-started/getting-started /docs/develop/getting-started/getting-started 301
```
Comment (nghi-ly) [inline _redirects]: should line 476 be updated so it matches line 478 for consistency?

Comment (nghi-ly) [inline _redirects]:
```suggestion
/guides/getting-started/getting-started /docs/develop/getting-started 301
```

File: website/docs/docs/develop/getting-started/getting-set-up.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: Getting set up in dbt Cloud
+id: getting-set-up
+description: "Set up a sample project to get you started using dbt Cloud."
+sidebar_label: "Overview"
```
Comment (nghi-ly) [inline website/docs/docs/develop/getting-started/getting-set-up.md]:
```suggestion
sidebar_label: "About setup"
```

File: website/docs/docs/develop/getting-started/getting-started.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Getting started with dbt Cloud
 id: getting-started
+sidebar_label: "Overview"
```
Comment (nghi-ly) [inline website/docs/docs/develop/getting-started/getting-started.md]:
```suggestion
sidebar_label: "About getting started"
```

File: website/docs/docs/develop/getting-started/learning-more.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
 ---
-title: "Learning more"
+title: "Learn more about dbt"
```
Comment (nghi-ly) [inline website/docs/docs/develop/getting-started/learning-more.md]: i'm unsure if this page is necessary as i click around the left sidebar. the page is mostly a topic list  which the left sidebar already provides so it seems redundant, low-value. consider not including it.

File: website/docs/docs/develop/getting-started/learning-more/getting-started-dbt-core.md [hunk 5]
```diff
@@ -2,16 +2,16 @@
 title: "Getting started with dbt Core"
 id: getting-started-dbt-core
 description: "Connecting your warehouse to dbt Core using the CLI."
-sidebar_label: "Getting started with dbt Core"
+sidebar_label: "Getting started"
```
Comment (nghi-ly) [inline website/docs/docs/develop/getting-started/learning-more/getting-started-dbt-core.md]: this guide doesn't have an "Overview page" accompaniment and i think it reads all right.

File: website/docusaurus.config.js [hunk 6]
```diff
@@ -123,7 +123,7 @@ var siteSettings = {
             },
             {
               label: 'Guides',
-              to: '/guides/getting-started',
+              to: '/guides/best-practices',
```
Comment (nghi-ly) [inline website/docusaurus.config.js:126]: great catch, @runleonarun !

**Review by nghi-ly — COMMENTED** [review]
@runleonarun , looks amazing! left a few questions and suggs.

File: website/docs/docs/develop/getting-started/getting-set-up.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
...
+---
+
+In this guide, you will learn how to set up dbt Cloud using a sample project and one of the most common data platforms. You can select from [BigQuery](/docs/develop/getting-started/getting-set-up/setting-up-bigquery), [Databricks](/docs/develop/getting-started/getting-set-up/setting-up-databricks), [Redshift](/docs/develop/getting-started/getting-set-up/setting-up-redshift), or [Snowflake](/docs/develop/getting-started/getting-set-up/setting-up-snowflake). If you're not sure, then try [BigQuery](/docs/develop/getting-started/getting-set-up/setting-up-bigquery).
```
Comment (runleonarun) [inline website/docs/docs/develop/getting-started/getting-set-up.md]:
```suggestion
Learn how to set up dbt Cloud using a sample project and one of the most common data platforms. You can select from [BigQuery](/docs/develop/getting-started/getting-set-up/setting-up-bigquery), [Databricks](/docs/develop/getting-started/getting-set-up/setting-up-databricks), [Redshift](/docs/develop/getting-started/getting-set-up/setting-up-redshift), or [Snowflake](/docs/develop/getting-started/getting-set-up/setting-up-snowflake). If you're not sure, then try [BigQuery](/docs/develop/getting-started/getting-set-up/setting-up-bigquery). 
```

File: _redirects [hunk 8]
```diff
@@ -463,3 +463,18 @@ docs/contributing/documenting-a-new-adapter /docs/contributing/adapter-developme
...
+/guides/getting-started/building-your-first-project/schedule-a-job /docs/develop/getting-started/building-your-first-project/schedule-a-job 301
+/guides/getting-started/building-your-first-project/test-and-document-your-project /docs/develop/getting-started/building-your-first-project/test-and-document-your-project 301
+/guides/getting-started/create-a-project /docs/develop/getting-started/create-a-project 301
+/guides/getting-started/getting-set-up /docs/develop/getting-started/getting-set-up 301
+/guides/getting-started/getting-set-up/setting-up-bigquery /docs/develop/getting-started/getting-set-up/setting-up-bigquery 301
+/guides/getting-started/getting-set-up/setting-up-databricks /docs/develop/getting-started/getting-set-up/setting-up-databricks 301
+/guides/getting-started/getting-set-up/setting-up-redshift /docs/develop/getting-started/getting-set-up/setting-up-redshift 301
+/guides/getting-started/getting-set-up/setting-up-snowflake /docs/develop/getting-started/getting-set-up/setting-up-snowflake 301
+/guides/getting-started/getting-started /docs/develop/getting-started 301
+/guides/getting-started/learning-more /docs/develop/getting-started/learning-more 301
```
Comment (runleonarun) [inline _redirects]:
```suggestion
/guides/getting-started/learning-more /docs/develop/getting-started/using-jinja 301
```

File: website/sidebars.js [hunk 9]
```diff
@@ -10,38 +10,61 @@ const sidebarSettings = {
...
+                "docs/develop/getting-started/building-your-first-project/build-your-first-models",
+                "docs/develop/getting-started/building-your-first-project/test-and-document-your-project",
+                "docs/develop/getting-started/building-your-first-project/schedule-a-job",
+              ],
+            },
+            {
+              type: "category",
+              label: "Learning more",
+              items: [
+                "docs/develop/getting-started/learning-more",
```
Comment (runleonarun) [inline website/sidebars.js]:
```suggestion
```

File: website/docs/docs/develop/getting-started/getting-started.md [hunk 10]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Getting started with dbt Cloud
 id: getting-started
```
Comment (nghi-ly) [inline website/docs/docs/develop/getting-started/getting-started.md]:
```suggestion
id: getting-started-dbt-cloud
```

Comment (nghi-ly) [reply website/docs/docs/develop/getting-started/getting-started.md]: aligning with `id: getting-started-dbt-core` for "Getting started with dbt Core" so we have getting-started URLs for both dbt Core and Cloud

File: _redirects [hunk 11]
```diff
@@ -463,3 +463,18 @@ docs/contributing/documenting-a-new-adapter /docs/contributing/adapter-developme
...
+/guides/getting-started/building-your-first-project/schedule-a-job /docs/develop/getting-started/building-your-first-project/schedule-a-job 301
+/guides/getting-started/building-your-first-project/test-and-document-your-project /docs/develop/getting-started/building-your-first-project/test-and-document-your-project 301
+/guides/getting-started/create-a-project /docs/develop/getting-started/create-a-project 301
+/guides/getting-started/getting-set-up /docs/develop/getting-started/getting-set-up 301
+/guides/getting-started/getting-set-up/setting-up-bigquery /docs/develop/getting-started/getting-set-up/setting-up-bigquery 301
+/guides/getting-started/getting-set-up/setting-up-databricks /docs/develop/getting-started/getting-set-up/setting-up-databricks 301
+/guides/getting-started/getting-set-up/setting-up-redshift /docs/develop/getting-started/getting-set-up/setting-up-redshift 301
+/guides/getting-started/getting-set-up/setting-up-snowflake /docs/develop/getting-started/getting-set-up/setting-up-snowflake 301
+/guides/getting-started/getting-started /docs/develop/getting-started 301
+/guides/getting-started/learning-more /docs/develop/getting-started/getting-started-dbt-core 301
```
Comment (nghi-ly) [inline _redirects:477]: is line 477 intentional? all good if it is, just flagging in case

**Review by nghi-ly — COMMENTED** [review]
looks great! left 2 questions

File: _redirects (hunk 11 above)
Comment (runleonarun) [reply _redirects:477]: Yea if we remove that learning more "Overview" page, I feel like this is the best place to take people.

Comment (nghi-ly) [reply _redirects:477]: ah i see! makes sense

File: website/docs/docs/develop/getting-started/getting-started.md [hunk 12]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Getting started with dbt Cloud
-id: getting-started
+id: getting-started-dbt-cloud
```
Comment (nghi-ly) [inline website/docs/docs/develop/getting-started/getting-started.md:3]: should the _redirects and sidebar.js files be updated?

**Review by nghi-ly — COMMENTED** [review]
last open item

File: website/docs/docs/develop/getting-started/getting-started.md (hunk 12 above)
Comment (runleonarun) [reply website/docs/docs/develop/getting-started/getting-started.md:3]: Yea just saw failing tests after I commit that suggestion 😵‍💫

**Review by nghi-ly — APPROVED** [review]
:shipit:

**Pull request merged**